lib/rules/custom-event-name-casing.js

```diff
@@ -85,11 +85,6 @@ module.exports = {
             },
             OBJECT_OPTION_SCHEMA
           ]
-        },
```

Comment:
should we also remove the `anyof` keyword

Reply:
Oh yeah, I forgot that.